cloud-azure/src/main/java/com/sequenceiq/cloudbreak/cloud/azure/AzureInstanceConnector.java

```diff
@@ -66,12 +67,45 @@ public List<CloudVmInstanceStatus> start(AuthenticatedContext ac, List<CloudReso
         return statuses;
     }
 
+    @Override
+    public List<CloudVmInstanceStatus> startWithLimitedRetry(AuthenticatedContext ac, List<CloudResource> resources,
+            List<CloudInstance> vms, Long timeboundInMs) {
+        LOGGER.info("Starting vms on Azure: {}", vms.stream().map(CloudInstance::getInstanceId).collect(Collectors.toList()));
```

Comment:
Similarly to stop ... log the timeout value.

Reply:
Changed